src/pages/brain/[pageId].tsx

```diff
@@ -0,0 +1,240 @@
+} from "@chakra-ui/react"
+import BrainCard from "components/brain/BrainCard"
+import Layout from "components/common/Layout"
+import LinkPreviewHead from "components/common/LinkPreviewHead"
+import Section from "components/common/Section"
+import CategorySection from "components/explorer/CategorySection"
+import { GetStaticPaths } from "next"
+import { NotionAPI } from "notion-client"
+import { DiscordLogo, Globe, IconProps, TwitterLogo } from "phosphor-react"
+
+import { PropsWithChildren } from "react"
+import { NotionRenderer } from "react-notion-x"
+import "react-notion-x/src/styles.css"
+import { BrainCardData } from "types"
+
+type CustomPageLinkProps = {
+  href: string
+  children: any
+}
+
+type Links = {
+  name: string
+  url: string
+  icon?: React.ForwardRefExoticComponent<
+    IconProps & React.RefAttributes<SVGSVGElement>
+  >
+}
+
+const CustomLink = ({ href, children }: PropsWithChildren<CustomPageLinkProps>) => {
+  const linkId =
+    href.slice(1, 9) +
+    "-" +
+    href.slice(9, 13) +
+    "-" +
+    href.slice(13, 17) +
+    "-" +
+    href.slice(17, 21) +
+    "-" +
+    href.slice(21)
+
+  return (
+    <Link href={`/brain/${linkId}`} colorScheme={"blue"} fontWeight="medium">
+      {children.props.block.properties.title[0][0]}
+    </Link>
+  )
+}
+
+const Header = (props) => {
+  const tags = props?.block?.properties?.["`SJU"]?.[0]?.[0]
+    .split(",")
+    .filter((tag) => tag !== "")
+  const guildId = props?.block?.properties?.JqtI?.[0]?.[0]
+  const websiteURL = props?.block?.properties?.uRBq?.[0]?.[0]
+  const twitterURL = props?.block?.properties?.["ByX="]?.[0]?.[0]
+  const discordURL = props?.block?.properties?.["Uw?a"]?.[0]?.[0]
+  const isContentTypePage = props?.block?.properties?.KYWu?.[0]?.[0] === "content"
+
+  if (
+    (tags === undefined || tags.length === 0) &&
+    guildId === undefined &&
+    websiteURL === undefined &&
+    twitterURL === undefined &&
+    discordURL === undefined
+  )
+    return
+  if (isContentTypePage) return
+
+  const links: Array<Links> = [
+    { name: "Guild", url: guildId },
+    { name: "website", url: websiteURL, icon: Globe },
+    { name: "Twitter", url: twitterURL, icon: TwitterLogo },
+    { name: "Discord", url: discordURL, icon: DiscordLogo },
+  ].filter((link) => link.url !== undefined && link.url !== "")
+
+  links.map((link) => {
+    if (link.name === "Guild") return
+    link.url = !link.url.startsWith("http") ? "http://" + link.url : link.url
+  })
+
+  return (
+    <Section>
+      <Wrap justify="space-between" pb="8px">
+        <Wrap>
+          {links?.map((link, index) => (
+            <Link
+              key={index}
+              href={link.name !== "Guild" ? link.url : `/${link.url}`}
+              isExternal={link.name !== "Guild"}
+            >
+              <Button colorScheme="alpha" color="whiteAlpha.900" height={8}>
+                {link.name === "Guild" ? (
+                  <Img src="/guildLogos/logo.svg" w="16px" mr="8px"></Img>
```

Comment:
this should be made a component and set as the icon for Guild in the links array (without mr), and the condition should be removed from here